<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReosMeteoFranceAromeApiWidget</class>
 <widget class="QWidget" name="ReosMeteoFranceAromeApiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="6" column="0" colspan="7">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Subset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Zone</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="mZoneCombo"/>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="mRunCombo">
     <item>
      <property name="text">
       <string>Last</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Before last</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Resolution</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="5" colspan="2">
    <widget class="QProgressBar" name="mProgressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5" colspan="2">
    <widget class="QPushButton" name="mConnectButton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="mResolCombo"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Extent</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="mExtentButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Run</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>API Key file</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QToolButton" name="mApiKeyFileButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="3">
    <widget class="QLineEdit" name="mKeyFileLineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mZoneCombo</tabstop>
  <tabstop>mResolCombo</tabstop>
  <tabstop>mConnectButton</tabstop>
  <tabstop>mExtentButton</tabstop>
  <tabstop>mRunCombo</tabstop>
  <tabstop>mKeyFileLineEdit</tabstop>
  <tabstop>mApiKeyFileButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
